libsemigroups_cppyy/froidure_pin.py

```diff
@@ -1,14 +1,13 @@
 """
 This file contains the interface to libsemigroups adapters; see
-
     https://libsemigroups.readthedocs.io/en/latest/_generated/libsemigroups__froidurepin.html
-
 for further details.
 """
 
 import cppyy
 import libsemigroups_cppyy.detail as detail
-
+import numpy as np
+from array import array
```

Comment:
Since there's no code in this file requiring `numpy` or `array`, please remove these two lines.